spec/features/sign_in_spec.rb

```diff
@@ -43,7 +43,7 @@
 
     # Redirect to DfE Signin and come back
     expect(page).to have_content("Sign out (#{user.first_name} #{user.last_name})")
-    expect(root_page).to be_displayed
+    expect(page.current_path).to eql("/rollover")
```

Comment:
I'm not sure I understand this change. The spec has a user with the default state ('rolled_over'). Given that the user has this state they should be directed to root_path not "/roll_over"?